src/components/tuTable/tuTableStore.ts

```diff
@@ -345,10 +389,59 @@ export class TuTableStore {
 				this.headerCount.value++;
 			}
 		});
+		if (localStorage.getItem(`table-${persistentId}`) !== null) {
+			const persistentColumns = JSON.parse(localStorage.getItem(`table-${persistentId}`));
+			const getNextHeader = (i: number) => {
+				const header = persistentColumns.columns[i].header;
+				let requiredHeader: TuHeaderDefn;
+				for (let j = 0; j < this.table.headers.length; j++) {
+					if (this.table.headers[j].field === header) {
+						requiredHeader = this.table.headers[j];
+						break;
+					}
+				}
+				return requiredHeader;
```

Comment:
Move the `return` statement and replace the `break` with it.
You dont need to use break and then return. Directly return